test/integration/IntegrationConfig.t.sol

```diff
@@ -488,6 +498,12 @@ contract IntegrationConfig is IntegrationDeployer, G2Operations, Constants {
             strategyCount = 2;
         } else if (strategyFlag == MANY) {
             strategyCount = _randUint({ min: 3, max: allStrats.length - 1 });
+        } else if (strategyFlag == FIFTEEN) {
+            strategyCount = 15;
+        } else if (strategyFlag == TWENTY) {
+            strategyCount = 20;
+        } else if (strategyFlag == TWENTYFIVE) {
+            strategyCount = 25;
```

Comment:
Were these added for the gas tests?

Reply:
Yup, tests seem to be ~12M gas for 200 operators and 25 strategies for updateOperators